### PR TITLE
Write page Your Account

### PR DESCRIPTION
**Pull request by magdeleine**
Base on fix/Bootstrap

### CODE REVIEW

**Comment by yannski** [comment]
Please use the gh-branch to base your work on

File: _posts/internals/2017-08-08-your-account.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+On the dashboard, you can see your account name at the top on the right. When you click, there is a menu with: [Profile](https://my.scalingo.com/profile), [SSH Keys](https://my.scalingo.com/keys), [Billing](https://my.scalingo.com/billing), [Referrals](https://my.scalingo.com/referrals), [Timeline](https://my.scalingo.com/timeline), [Logout](https://my.scalingo.com/#).
+
+### [Profile](https://my.scalingo.com/profile)
+
+Here, you can change your [Basic Informations](https://my.scalingo.com/profile) like Username, First Name, Last Name, Location and Company. You can also subscibe to our newsletter or unsubscribe.
+
+There is a section to [change your Email](https://my.scalingo.com/profile). 
+
+You will find your [API Token](https://my.scalingo.com/profile) for our Platform and you can regenerate it.
```
Comment (yannski) [inline _posts/internals/2017-08-08-your-account.md]: Mention the purpose of the token (API)

File: _posts/internals/2017-08-08-your-account.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Another section is about [Integrations](https://my.scalingo.com/profile) like GitHub.
+
+Finally you will find the button to delete your account.
```
Comment (yannski) [inline _posts/internals/2017-08-08-your-account.md]: Mention how to change password

File: _posts/internals/2017-08-08-your-account.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+### [SSH Keys](https://my.scalingo.com/keys)
+
+It is in that page that you can import or delete a SSH key.
```
Comment (yannski) [inline _posts/internals/2017-08-08-your-account.md]: Mention the purpose of the SSH keys

**Comment by Soulou** [comment]
screenshot

**Pull request merged**